test-reports/clover.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@Emmanuelaaron Why do we have this files tracked?

Reply:
I have removed it